app/code/Magento/Developer/Model/View/Asset/PreProcessor/PreprocessorStrategy.php

```diff
@@ -42,32 +42,34 @@ class PreprocessorStrategy implements PreProcessorInterface
     private $state;
 
     /**
-     * Constructor
-     *
      * @param AlternativeSourceInterface $alternativeSource
      * @param FrontendCompilation $frontendCompilation
      * @param ScopeConfigInterface $scopeConfig
+     * @param State|null $state
      */
     public function __construct(
         AlternativeSourceInterface $alternativeSource,
         FrontendCompilation $frontendCompilation,
-        ScopeConfigInterface $scopeConfig
+        ScopeConfigInterface $scopeConfig,
+        ?State $state = null
```

Comment:
please make this argument not optional as this class isn't marked with api tag

Reply:
Hmm, isn't it? https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Developer/Model/View/Asset/PreProcessor/PreprocessorStrategy.php#L19

Reply:
Ah. sorry, I missed it